src/Compute/Compute.Test/ScenarioTests/VirtualMachineTests.ps1

```diff
@@ -3132,6 +3132,66 @@ function Test-VirtualMachineGetStatusWithHealhtExtension
     }
 }
 
+<#
+.SYNOPSIS
+Test Virtual Machines's Status With Health Extension
+Description:
+This test creates a virtual machine and adds a vm health extension.It then removes the health Extension and tests Remove-AzVMExtension
+#>
+function Test-VirtualMachineRemoveExtension
+{
+    # Setup
+    $rgname = Get-ComputeTestResourceName
+
+    try
+    {
+        # Common
+        $loc = Get-ComputeVMLocation;
+        $loc = $loc.Replace(' ', '');
+
+        New-AzResourceGroup -Name $rgname -Location $loc -Force;
+        
+        # VM Profile & Hardware
+        $vmname = 'vm' + $rgname;
+
+        # OS & Image
+        $username = "admin01";
+        $password = $PLACEHOLDER | ConvertTo-SecureString -AsPlainText -Force;
+        $cred = new-object -typename System.Management.Automation.PSCredential -argumentlist $username, $password;
+        [string]$domainNameLabel = "vcrptestps7691-6f2166";
```

Comment:
don't need this [string]

Reply:
don't hardcode the domainnamelabel, rather have something like 'd' + $rgname

Reply:
Cool, makes sense